Always use a 64K receive buffer in the UDP listener (#6470)
DataDog/datadog-agent: docs/dogstatsd/configuration.md

## `dogstatsd_buffer_size`

> The `dogstatsd_buffer_size` parameter configures how many bytes maximum the PacketAssembler should put into one packet.

Reviewer: ```suggestion
The `dogstatsd_buffer_size` parameter configures the maximum number of bytes the PacketAssembler should put into one packet.
```

> Please note that increasing this buffer size, together with `dogstatsd_queue_size`, has a huge impact on the maximum memory usage:

Reviewer: ```suggestion
**Note**: Increasing this buffer size, together with `dogstatsd_queue_size`, has a huge impact on the maximum memory usage:
```

This parameter represents how many packet sets flushed from the packets buffer to the parser could be
buffered. The idea is to read as fast as possible on the socket and to store packets here if the rest

> of the pipeline is having slow-down for any reasons. Each packet is of size `dogstatsd_buffer_size`.

Reviewer: ```suggestion
of the pipeline is slowing down for any reason. Each packet is of size `dogstatsd_buffer_size`.
```
